state/sqlserver/migration.go

```diff
@@ -159,38 +160,39 @@ func (m *migration) ensureIndexedPropertyExists(ix IndexedProperty, db *sql.DB)
 		m.store.tableName,
 		ix.ColumnName)
 
-	return runCommand(tsql, db)
+	return runCommand(ctx, db, tsql)
 }
 
 /* #nosec. */
-func (m *migration) ensureDatabaseExists(db *sql.DB) error {
+func (m *migration) ensureDatabaseExists(ctx context.Context, db *sql.DB) error {
 	tsql := fmt.Sprintf(`
 IF NOT EXISTS (SELECT * FROM sys.databases WHERE name = N'%s')
 	CREATE DATABASE [%s]`,
 		m.store.databaseName, m.store.databaseName)
 
-	return runCommand(tsql, db)
+	return runCommand(ctx, db, tsql)
 }
 
 /* #nosec. */
-func (m *migration) ensureSchemaExists(db *sql.DB) error {
+func (m *migration) ensureSchemaExists(ctx context.Context, db *sql.DB) error {
 	tsql := fmt.Sprintf(`
 	IF NOT EXISTS(SELECT * FROM sys.schemas WHERE name = N'%s')
 		EXEC('CREATE SCHEMA [%s]')`,
 		m.store.schema, m.store.schema)
 
-	return runCommand(tsql, db)
+	return runCommand(ctx, db, tsql)
 }
 
 /* #nosec. */
-func (m *migration) ensureTableExists(db *sql.DB, r migrationResult) error {
+func (m *migration) ensureTableExists(ctx context.Context, db *sql.DB, r migrationResult) error {
 	tsql := fmt.Sprintf(`
 	IF NOT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = '%s' AND TABLE_NAME = '%s')
     	CREATE TABLE [%s].[%s] (
 			[Key] 			%s CONSTRAINT PK_%s PRIMARY KEY,
 			[Data]			NVARCHAR(MAX) NOT NULL,
-			[InsertDate] 	DateTime2 NOT NULL DEFAULT(GETDATE()),
-			[UpdateDate] 	DateTime2 NULL,`,
+			[InsertDate] 	DateTime2 NOT NULL DEFAULT(CURRENT_TIMESTAMP),
```

Comment:
Is this a breaking change? `GETDATE()` to `CURRENT_TIMESTAMP`?

Reply:
Reverted.